styles.css

```diff
@@ -0,0 +1,100 @@
+/* Setting a base font size of 10px give us easier rem calculations
+         Info: 1rem === 10px, 1.5rem === 15px, 2rem === 20px and so forth
+       */html {
```

Comment:
you should add space before and after the comments. When you add linters it will give you errors about this.